bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/ObserveTimeLineNode.java

```diff
@@ -90,20 +91,45 @@ public static long getDuration(PlannedTime pt, int step) {
         } else {
             buf.append("<p align=\"center\"><b>").append(SequenceTabUtil.shortDatasetLabel(plannedTime.sequence.getStep(step))).append("</b></p>");
 
-            buf.append("<table><tr><th>Event</th><th>Sec</th></tr>");
+            buf.append("<table><tr><th colspan=\"2\">Event</th><th colspan=\"2\">Sec</th></tr>");
```

Comment:
Sorry you had to resort to this 😄 
Using html in swing controls is quite fragile though often is the only way around it. This changes probably need to be tested across platforms though

Reply:
Ugly, I know 😳. I can test the OT on Linux, are you worried this won't be displayed properly on Windows?

Reply:
That's a good point. Recently we discovered some swing issues under windows 10, so this will need to be tested. I don't see anything that raises a flag, but if we can make a version of this for Linux/OSX/Windows and let Bryan try it on the three platforms, that would remove the concern.

Reply:
Alright I have a spare Windows 10 license, I'll use it in a VM to it. I guess we'll need more Windows 10 testing in the future.

Reply:
I tested it Windows and Linux. In Arch Linux with GNOME, it looks fine. In Windows 10 all the fonts look very pixelated (maybe it's because of my Remote Desktop compression settings or something), but aside of that, the alignment of the columns for the tooltip looks alright.